source/agora/node/Node.d

```diff
@@ -387,9 +397,81 @@ public class Node : API
         }
     }
 
+    /***************************************************************************
+
+        Returns an instance of a EnrollmentManager
+
+        Params:
+            data_dir = path to the data dirctory
+            node_config = the node config
+
+        Returns:
+            the enrollment manager
+
+    ***************************************************************************/
+
+    protected EnrollmentManager getEnrollmentManager (string data_dir,
+        in NodeConfig node_config, Ledger ledger, UTXOSet utxo_set)
+    {
+        return new EnrollmentManager(buildPath(data_dir, "validator_set.dat"),
+            node_config.key_pair, ledger, utxo_set);
+    }
+
+    /***************************************************************************
+
+        Get UTXOs from the UTXO set
+
+        Params:
+            address = used to identify the owner of UTXOs
+
+        Returns:
+            the associative array for UTXOs
+
+    ***************************************************************************/
+
+    protected UTXOSetValue[Hash] getUTXOs (PublicKey address) @safe
```

Comment:
Since this function is only forwarding, and it's protected, you can remove it.
Instead call `this.utxo_set.getUTXOs` directly.

But I realize that most members of `Node` are `private`. Perhaps they should be made `protected`. Could you do this in another commit / PR (since it's a no brainer, another PR would get merged very quickly).

Reply:
Sure, I will do that in another commit of this PR.